docs/upgrades/unreleased.md

```diff
@@ -382,3 +382,102 @@ If you plan to use the Angular SDK with XMCloud, you will need to perform next s
 * `express` dependency is marked as a peer dependency, so you need to match the required version "^4.19.2".
 
 * Copy the `.gitignore` file from fresh latest version proxy app into your existing proxy app, if not already present.
+
+# Nextjs - XMCloud
```

Comment:
This entry should not be added here, since it's a part of 22.2.0 release, while current document is for next release (22.3.0 may be)

Reply:
both are still unreleased and both belong here. I will use these latest entries from `nextjs-xmcloud` for doc input before 22.2 release, and will remove them from `unreleased` after doing so.

Reply:
You can just create a new document under 22.x folder